### PR TITLE
Fix GET operation response, unmarshal correct KeyValue type

### PR DESCRIPTION
**Pull request by baum**
Fix GET operation response, unmarshal correct KeyValue type

See sample test code:

* https://github.com/baum/kmip-go/blob/master/ciphertrust_test.go

### CODE REVIEW

**Comment by humblec** [comment]
Cc @ansel1 @jskazinski

File: base_objects.go [hunk 1]
```diff
@@ -125,13 +125,11 @@ type AttestationCredentialValue struct {
...
-// TODO: Unmarshaler impl which unmarshals correct KeyValue type.
+
 type KeyBlock struct {
-	KeyFormatType      kmip14.KeyFormatType
-	KeyCompressionType kmip14.KeyCompressionType `ttlv:",omitempty"`
-	// KeyValue should be either []byte or KeyValue
-	KeyValue               interface{}                   `ttlv:",omitempty"` // should be either a []byte or KeyValue
+	KeyFormatType          kmip14.KeyFormatType
+	KeyCompressionType     kmip14.KeyCompressionType     `ttlv:",omitempty"`
+	KeyValue               KeyValue                      `ttlv:",omitempty"`
```
Comment (ansel1) [inline base_objects.go:132]:
```suggestion
	KeyValue               *KeyValue                      `ttlv:",omitempty"`
```

In the 1.4 spec, KeyValue is optional.

**Comment by ansel1** [comment]
Could some simple marshal/unmarshal tests be added, similar to the TestTemplateAttribute_marshal() test?  It just builds raw  TTLV values, then tests that the marshal/unmarshal roundtrip doesn't alter them.

File: base_objects.go (hunk 1 above)
Comment (baum) [reply base_objects.go:132]: LGTM 🖖

**Comment by baum** [comment]
> Could some simple marshal/unmarshal tests be added, similar to the TestTemplateAttribute_marshal() test? It just builds raw TTLV values, then tests that the marshal/unmarshal roundtrip doesn't alter them.

[Added](https://github.com/ThalesGroup/kmip-go/pull/19/commits/88b0550cc8d6cbeb7d080e5192f0bd4d4d1ed7e8) TestGetResponsePayload_unmarshal/SymmetricKey

**Pull request merged**